src/DssSpell.sol

```diff
@@ -16,35 +16,22 @@
 
 pragma solidity 0.6.12;
 
+import {Fileable, ChainlogLike} from "dss-exec-lib/DssExecLib.sol";
```

Comment:
any particular reason to use ChainlogLike from dss-exec-lib instead of ChainlogAbstract in dss-interfaces? To reduce code?